tools/manifest.sh

```diff
@@ -10,11 +10,11 @@ cd "$(dirname "$0")"/..
 #    ./tools/manifest.sh [PACKAGE [VERSION_REQ]]
 
 if [[ $# -gt 0 ]]; then
-    cargo run --manifest-path tools/codegen/Cargo.toml --release -- "$@"
+    cargo +nightly run --manifest-path tools/codegen/Cargo.toml --release -- "$@"
```

Comment:
I know that the manifest CI has been using nightly, just curious why nightly is used cc @taiki-e 

If it is switched to stable, then I can cache it completely using [action Swatinem/rust-cache](https://github.com/Swatinem/rust-cache), saving about 30s.

Reply:
It doesnt compile with stable.

Reply:
Thanks, I see the use of `LazyCell` makes it a problem.

IIRC it's going to be stablised in 1.80, and I think it makes more sense to switch to stable so that it can be cached.